doc/ProvisionReleaseBuild.md

```diff
@@ -1,28 +1,38 @@
-# Provision Release Build
+# Provision & Deploy Cortx Build Stack
 
 You will need to complete this [guide](https://github.com/Seagate/cortx/blob/main/doc/Release_Build_Creation.rst) before moving onto the steps below.
 
-### 1.  Change Hostname
+### Pre-requisite
+
+- Change Hostname by running,
    ```
    sudo hostnamectl set-hostname deploy-test.cortx.com
    ```
-   - Please use this hostname to make reduce issues further on in the process.
+   - Please use this hostname to avoid issues further in the bootstrap process.
    - Make sure the hostname is changed by running `hostname -f`
-  
-### 2. Disable SElinux
-
-```
-sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
-```
-- Reboot the VM  by running `reboot` 
-
-### 3. Install Provisioner API
-   
-   - Set repository URL
+ 
+- Disable SElinux by running,
+   ```
+   sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
+   ```
+- Set repository URL
    ```
    export CORTX_RELEASE_REPO="file:///var/artifacts/0"
    ```   
-   - Install Provisioner API and requisite packages
+- Reboot your VM by running `reboot` command
+
+- Cleanup temporary repos
```

Comment:
This should be after https://github.com/Seagate/cortx/blob/mukul-seagate11-patch-12/doc/ProvisionReleaseBuild.md#1-install-provisioner-api-and-requisite-packages

Reply:
done